lib/screens/create_region_event_screens/add_region_event_image.dart

```diff
@@ -1,10 +1,83 @@
 import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:seeds/components/flat_button_long.dart';
+import 'package:seeds/components/select_picture_box/select_picture_box.dart';
+import 'package:seeds/design/app_colors.dart';
+import 'package:seeds/design/app_theme.dart';
+import 'package:seeds/domain-shared/event_bus/event_bus.dart';
+import 'package:seeds/domain-shared/event_bus/events.dart';
+import 'package:seeds/domain-shared/page_command.dart';
+import 'package:seeds/domain-shared/ui_constants.dart';
+import 'package:seeds/screens/create_region_event_screens/interactor/viewmodels/create_region_event_bloc.dart';
+import 'package:seeds/utils/build_context_extension.dart';
 
 class AddRegionEventImage extends StatelessWidget {
   const AddRegionEventImage({Key? key}) : super(key: key);
 
   @override
   Widget build(BuildContext context) {
-    return Scaffold(appBar: AppBar(title: const Text("Create Event")));
+    return BlocConsumer<CreateRegionEventBloc, CreateRegionEventState>(
+        listenWhen: (previous, current) => current.pageCommand != null || previous.file != current.file,
```

Comment:
|| previous.file != current.file, Are you sure about this?

Reply:
yes is for the Navigator.of(context).pop(); to remove the authentication screen